### ISSUE TITLE
Do you write  UnFlow using pytorch?

### ISSUE THREAD

**Issue by hzk7287** [simonmeister/UnFlow#42]
Hi, I like your paper very much. I would like to use pytorch to build my model. But I only find tensorflow 
version.  If I want to use this loss function in my pytorch model, should I write the loss function by myself?  

Thank you in advance

**Comment by simonmeister** [comment]
Thanks! Yes, I don't have time to write a port unfortunately.

**Comment by sniklaus** [comment]
In case anyone is interested, I reimplemented the inference part in PyTorch: https://github.com/sniklaus/pytorch-unflow

Huge thanks Simon for your work, I enjoyed reading your paper and am happy to see you made the implementation publicly available!

**Comment by hzk7287** [comment]
Many thanks！

**Comment by simonmeister** [comment]
Thanks! I pointed to your repo in the README.

**Comment by sniklaus** [comment]
Thank you, I greatly appreciate it!

**Comment by harikiran2995** [comment]
> Huge thanks Simon for your work, I enjoyed reading your paper and am happy to see you made the implementation publicly available!

can i train?
